sheeprl/utils/callback.py

```diff
@@ -105,14 +105,14 @@ def _ckpt_rb(
         """
         if isinstance(rb, ReplayBuffer):
             # clone the true done
-            state = rb["dones"][(rb._pos - 1) % rb.buffer_size, :].copy()
+            state = rb["truncated"][(rb._pos - 1) % rb.buffer_size, :].copy()
             # substitute the last done with all True values (all the environment are truncated)
-            rb["dones"][(rb._pos - 1) % rb.buffer_size, :] = True
+            rb["truncated"][(rb._pos - 1) % rb.buffer_size, :] = 1
         elif isinstance(rb, EnvIndependentReplayBuffer):
             state = []
             for b in rb.buffer:
-                state.append(b["dones"][(b._pos - 1) % b.buffer_size, :].copy())
-                b["dones"][(b._pos - 1) % b.buffer_size, :] = True
+                state.append(b["truncated"][(b._pos - 1) % b.buffer_size, :].copy())
```

Comment:
Are we sure that every algorithm sets in the buffer the `truncated` flag? For example PPORecurrent just sets the done in the buffer